gnome-extra/cinnamon-desktop/cinnamon-desktop-4.8.1.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+inherit meson gnome2-utils python-any-r1 xdg
+
+DESCRIPTION="A collection of libraries and utilites used by Cinnamon"
+HOMEPAGE="https://projects.linuxmint.com/cinnamon/"
+SRC_URI="https://github.com/linuxmint/cinnamon-desktop/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+ FDL-1.1+ LGPL-2+"
+SLOT="0/4" # subslot = libcinnamon-desktop soname version
+KEYWORDS="~amd64 ~arm64 ~x86"
+
+COMMON_DEPEND="
+	>=dev-libs/glib-2.37.3:2[dbus]
+	>=dev-libs/gobject-introspection-0.10.2:=
+	>=gnome-base/gsettings-desktop-schemas-3.5.91
+	media-sound/pulseaudio[glib]
+	sys-apps/accountsservice
+	x11-libs/cairo:=[X]
+	>=x11-libs/gdk-pixbuf-2.22:2[introspection]
+	>=x11-libs/gtk+-3.3.16:3[introspection]
+	x11-libs/libX11
+	>=x11-libs/libXext-1.1
+	x11-libs/libxkbfile
+	>=x11-libs/libXrandr-1.3
+	x11-misc/xkeyboard-config
+"
+RDEPEND="${COMMON_DEPEND}"
```

Comment:
Looks like `COMMON_DEPEND` is useless here. Just define it as `RDEPEND` and use it in `DEPEND`.